docs/getting-started/install-endpoint.asciidoc

```diff
@@ -26,46 +26,56 @@ image::images/install-endpoint/security-integration.png[Search result for "Endpo
+If you have upgraded to an {stack} version that includes {fleet-server} (>=7.13.0), you will need to redeploy your agents. Review the following scenarios to ensure you take the appropriate steps.
 
 * If you redeploy the {agent} to the same machine through the {fleet} application after you upgrade, a new agent will appear.
-* If you want to remove the {agent} entirely without transitioning to the {fleet-server}, then you will need to manually uninstall the agent on the machine. This will also uninstall the endpoint. See {fleet-guide}/uninstall-elastic-agent.html[Uninstall Elastic Agent].
-* In the rare event that the {agent} fails to uninstall, you might need to manually uninstall the endpoint. See <<uninstall-endpoint, Uninstall an endpoint>> at the end of this topic.
+* If you want to remove the {agent} entirely without transitioning to the {fleet-server}, then you will need to manually uninstall the agent on the machine. This will also uninstall the endpoint. Refer to {fleet-guide}/uninstall-elastic-agent.html[Uninstall Elastic Agent].
+* In the rare event that the {agent} fails to uninstall, you might need to manually uninstall the endpoint. Refer to <<uninstall-endpoint, Uninstall an endpoint>> at the end of this topic.
 
-NOTE: Endpoint Security cannot be integrated with an {agent} in Standalone mode.
+[discrete]
+[[enroll-agent]]
+=== Add the {agent}
 
 . Go to *{fleet}* -> *Agents* -> **Add agent**.
 +
 [role="screenshot"]
 image::images/install-endpoint/add-agent.png[Add agent flyout on the Fleet page.]
+
+. Select an agent policy for the {agent}. You can select an existing custom policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
 +
-. In the Add agent pane, download the {agent} on your host's machine.
-. After the download is complete, select the **Endpoint Security** integration.
+The selected policy should include {endpoint-sec}.
 +
 [role="screenshot"]
-image::images/install-endpoint/endpoint-configuration.png[Detail of Add agent flyout with Endpoint Security integration highlighted.]
-+
-. After the {agent} is installed on your host machine, open a command-line interface and navigate to the directory where the agent is located. Copy the commands from {fleet} for your OS to enroll and run the agent.
-+
-. After you have enrolled the {agent} on your host, select **Close**. The host now appears on the **Endpoints** page in the {security-app}.
+image::images/install-endpoint/endpoint-configuration.png[Add agent flyout with Endpoint Security integration highlighted.,575]
+
+. Download and extract the {agent} package on your host's machine.
+
+. Open a command-line interface and navigate to the directory where the {agent} is located. Copy and run the command from {fleet} for your OS to enroll and run the agent.
```

Comment:
```suggestion
. Open a command-line interface and navigate to the directory where the {agent} is located. Copy and run the command from {fleet} for your OS to enroll and run the {agent}.
```

Reply:
Might be a bit repetitive since the full name is already mentioned in the previous sentence, so feel free to ignore this recommendation if it makes the second sentence unnecessarily wordy.